Makefile

```diff
@@ -38,7 +38,7 @@ proto:
 bash:
 	$(DC_RUN_PHP) bash
 style:
-	$(DC_RUN_PHP) php ./vendor/bin/php-cs-fixer fix --config=.php-cs-fixer.php --stop-on-violation --using-cache=no -vvv
+	$(DC_RUN_PHP) php ./vendor/bin/php-cs-fixer fix --config=.php-cs-fixer.php --using-cache=no -vvv
```

Comment:
Unrelated, but this was annoying me as you needed to run `make style` multiple times just to fix everything.

Reply:
No problem.  Happy this helps with the developer experience!